janusgraph-solr/src/main/java/org/janusgraph/diskstorage/solr/SolrIndex.java

```diff
@@ -205,6 +216,16 @@ public static Mode parse(String mode) {
             "When mutating - wait for the index to reflect new mutations before returning. This can have a negative impact on performance.",
             ConfigOption.Type.LOCAL, false);
 
+    
+    /** Security Configuration */
+    
+    public static final ConfigOption<Boolean> KERBEROS_ENABLED = new ConfigOption<Boolean>(SOLR_NS,"kerberos-enabled",
+            "Whether SOLR instance is Kerberized or not.",
+            ConfigOption.Type.GLOBAL_OFFLINE, false);
```

Comment:
I'm not sure how valuable this parameter is if it's configured as `GLOBAL_OFFLINE`. Do users typically toggle Kerberos on and off? If not, it seems like you could just use `KERBEROS_CONFIG` as the only config parameter for whether to use Kerberos or not.

Reply:
> Do users typically toggle Kerberos on and off?
No I would say that is not a typical use case.

Reply:
If it makes sense to support changing the `KERBEROS_ENABLED` setting per connection (for example, if Solr or HBase support it also), then you should declare it as `ConfigOption.Type.MASKABLE` rather than `ConfigOption.Type.GLOBAL_OFFLINE`.

Reply:
After discussion I will make the change to `MASKABLE`.